public/sw.js

```diff
@@ -0,0 +1,231 @@
+const OFFLINE_VERSION = 1;
```

Comment:
Is this variable needed?

Reply:
This is really just used for tracking the versioning of the different caches, but in reality no it is not needed. I'll remove it for now.